watertap/unit_models/tests/test_gac.py

```diff
@@ -52,11 +53,16 @@
 )
 from watertap.costing import WaterTAPCosting
 
-
 __author__ = "Hunter Barber"
 
 solver = get_solver()
 
+# inputs for badly_scaled_var_generator used across test frames
+sv_large = 1e2
+sv_small = 1e-2
```

Comment:
Seems more stringent than necessary, but that's ok. Did you have to be this stringent for things to work?

Reply:
The model works fine under the defaults! Something I changed, in addition to having dynamic scaling factors for variables as a function of the default sf for `H2O` and `target_species` molar flow from the property package, dynamic scaling factors according to inlet volumetric flow because it was more convenient for variables such as bed volume. So I used tight bounds when testing these scaling factors just because I wanted to make sure I was actually doing it right and the default of 1e4 (or whatever the default is) is fine for edge cases, but I wanted to be more nearer to 1 for a traditional process scale such as this.